resources/prod/config.yml

```diff
@@ -0,0 +1,48 @@
+#
+# Configuration file for Docker registry
+# https://distribution.github.io/distribution/about/deploying/
+#
+# Before using, notifications/endpoints/headers/Authorization must be filled in
+#
+version: 0.1
+log:
+    level: info
+    formatter: text
+    fields:
+        service: registry
+http:
+    addr: :443
+    tls:
+        certificate: /etc/docker/registry/ssl/public.crt
```

Comment:
shouldn't this be `/etc/docker/registry/ssl/certificate.pem`?

Reply:
Yes, good catch!  The path was correct in dev' but wrong in prod'.  I just went back and compared the dev' and prod' files to make sure there was nothing else broken.